classes/class-bsf-custom-fonts-render.php

```diff
@@ -139,6 +139,39 @@ public function __construct() {
 			add_filter( 'elementor/fonts/additional_fonts', array( $this, 'add_elementor_fonts' ) );
 			// Astra filter before creating google fonts URL.
 			add_filter( 'astra_google_fonts_selected', array( $this, 'remove_custom_font_google_url' ) );
+			// Hook the enqueue style method to the 'wp_enqueue_scripts' action.
+			add_action( 'wp_enqueue_scripts', array( $this, 'my_custom_fonts_enqueue_style' ) );
```

Comment:
```suggestion
			add_action( 'wp_enqueue_scripts', array( $this, 'custom_fonts_frontend_styles' ) );
```

Reply:
Anyways, we don't need this action & code, because the same operation we are doing above on L-135

Reply:
@imnavanath You mean should I remove the entire add action?

Reply:
yes with its function

Reply:
Okay function & action removed, Thanks for pointing out.